src/Item.js

```diff
@@ -71,6 +109,27 @@ function Item({ userToken, item }) {
     setChecked(!checked);
   };
 
+  // [delete-item] 4a. If yes, then remove from firestore database
+  const handleDelete = () => {
+    setOpenModal(true);
+    db.collection(userToken)
+      .doc(id)
+      .delete()
+      .then(() => {
+        console.log('it worked!');
```

Comment:
Could use your snackbar that's already installed to send a success message.

Reply:
Great point!